slidedeck/src/components/reducer/ReducerExample.jsx

```diff
@@ -9,12 +9,10 @@ const agents = ['Sterling Archer', 'Lana Kane', 'Cyril Figgis'];
 const agentReducer = (state = initialState, action) => {
   switch (action.type) {
     case 'ADD_EMPLOYEE':
-      return Object.assign({}, state, {
-return {
-  ...state, 
-  agents: state.agents.concat(action.payload)
-}
-      });
+      return {
```

Comment:
What the heck happened? My bad. 😊

Reply:
Oh, I see. Github doesn't support multi-line PR suggestions without some gymnastics involved. https://github.community/t5/How-to-use-Git-and-GitHub/PR-multiline-review/m-p/6779#M2261